tests/integration_tests/performance/test_boottime.py

```diff
@@ -184,3 +197,11 @@ def test_boottime(
         vm.start()
         boottime_us = _get_microvm_boottime(vm)
         metrics.put_metric("boot_time", boottime_us, unit="Microseconds")
+        timestamps = find_events(vm.log_data)
+        build_time = timestamps["build microvm"]["duration"]
+        metrics.put_metric("build_time", build_time.microseconds, unit="Microseconds")
+        metrics.put_metric(
+            "guest_boot_time",
```

Comment:
is the notation different on purpose (`build_time` and `guest_boot_time`), not `guest_build_time` or `boot_time`?

Reply:
yeah, that would be at least consistent

Reply:
What I wanted to convey is that guest_boot_time depends on the guest OS, while build time is Firecracker/host.

Reply:
ok, that makes sense indeed.

Reply:
we don't need a changelog entry for this, do we?

Reply:
Yeah I don't think we do. I don't want to publicize it too much yet :)